### DISCUSSION TITLE
Trying to change default logo

### DISCUSSION THREAD

**Discussion by fenix2222** [blockscout/blockscout#9121] [Q&A] [answered]
We are using v6.0

I have set environment var for the logo and also made sure image is rebuilt, I can see in docker env variable is correctly set, however, logo in the menu is still references from some sprite.svg, and if I check deployed files, there is no "images" folder there

I am using docker version and follwoing this readme: https://docs.blockscout.com/for-developers/information-and-settings/env-variables#header-management

<img width="1344" alt="image" src="https://github.com/blockscout/blockscout/assets/6656756/41785945-203d-41d0-89b5-3b2bd2058b62">

**Comment by GimluCom** [comment]
If you are using the new UI, please check you have set these ENV variables
```env
NEXT_PUBLIC_NETWORK_LOGO=https://placekitten.com/240/40
NEXT_PUBLIC_NETWORK_ICON=https://placekitten.com/60/60
```

**Comment by fenix2222** [comment]
Thanks, is there a similar one for favicon?

Also, is there any documentation for this?

**Reply by GimluCom** [reply, marked as answer]
Yes, you can find it at https://github.com/blockscout/frontend/blob/main/docs/ENVS.md#favicon
